### PR TITLE
PALS-1210: Add Ethnio survey to homepage, item detail,  search results and image group

### PR DESCRIPTION
**Pull request by ymouzakis**
Resolves PALS-1210

## Description

We want to add the new Ethnio survey to homepage, item detail,  search results and image group.

## Checks

**Have you written any necessary unit tests?**

- [ ] Yes
- [ ] Not yet
- [ ] Not applicable

**Have you added or updated any necessary integration tests?**

- [ ] Yes
- [ ] Not yet
- [ ] Not applicable

**Browsers tested on:**

- [ ] Chrome
  - [ ] Mobile
- [ ] Safari
  - [ ] Mobile
- [ ] Firefox
  - [ ] Mobile
- [ ] Edge
- [ ] IE11
- [ ] Not applicable

### CODE REVIEW

**Review by michael-iden — APPROVED** [review]

File: src/app/app.component.ts [hunk 1]
```diff
@@ -264,6 +265,8 @@ export class AppComponent {
   }
 
   ngOnInit() {
+    // Ethnio survey 11-10-21
+    this._script.loadScript('ethnio-survey');
```
Comment (NazimHAli) [inline src/app/app.component.ts:269]: This will be added to every page - recommend removing it. I don't think it needs to be added anywhere else.

Comment (ymouzakis) [reply src/app/app.component.ts:269]: Ah I see....How would I add it to the item detail page @NazimHAli ?

Comment (NazimHAli) [reply src/app/app.component.ts:269]: It's already included here : https://github.com/ithaka/aiw-ui/pull/1362/files#diff-416072271f1854db0d48a9b6df3b354d3e35c12f68083ceebe4017a03e5db046L235

Comment (NazimHAli) [reply src/app/app.component.ts:269]: Does it not show up after you deployed it to test?

**Comment by ymouzakis** [comment]
I accidentally merged to stage before removing the survey from  src/app/app.component.ts. Made PALS-1210-2 to fix it.

**Pull request merged**